pep-0604.rst

```diff
@@ -39,15 +37,13 @@ The verbosity of this syntax does not help the adoption.
 Proposal
 ========
 
-Inspired by Scala language [5]_, this proposal adds operator ``__or__()`` in the root ``type``.
-With this new operator, it is possible to write ``int | str`` in place of ``Union[int,str]``.
-This proposition uses the standard meaning of the ``|`` operator.
-Then, it is possible to extend ``isinstance()`` and ``issubclass()``
-to accept this new syntax:
-
-::
+Inspired by Scala language [2]_ and Pike [3]_, this proposal adds operator ``type.__or__()``.
```

Comment:
I think that `str.__or__` and `NoneType.__or__` will also need adding so that `Union["forwardRef", type]` and `Union[None, type]` can also be converted to `"forwardref" | type` and `None | type` respectively

Reply:
No, that can be done using `type.__ror__()`.

(PS: I plan to review this, but I've got a big backlog.)